components/automate-ui/README.md

```diff
@@ -55,24 +55,17 @@ Certain packages in package.json are constrained for the reasons detailed here.
 At any future moment though, the reasons for constraint here could be invalidated, so this should be updated as needed when package.json is updated.
 While it is problematic to document this information due to the maintenance burden, the value of having this in one place outweighs that burden.
 
```

Comment:
Notice that `npm outdated` is showing these:
```
Package                Current    Wanted       Latest  Location
@ngrx/effects           10.1.0    10.1.2       10.1.2  automate-ui
@ngrx/entity            10.1.0    10.1.2       10.1.2  automate-ui
@ngrx/router-store      10.1.0    10.1.2       10.1.2  automate-ui
@ngrx/store             10.1.0    10.1.2       10.1.2  automate-ui
@ngrx/store-devtools    10.1.0    10.1.2       10.1.2  automate-ui
```
If you really want to constrain these, as you have mentioned, so it does not increment even the last segment (the patch number), you need to update package.json to use exact version (e.g. `10.1.0` instead of `^10.1.0`). Otherwise, IIRC, if you just run `npm update` right now, it will bump up to the "wanted" version. (You could try to see if that is true.).
Also, consider adding an entry for these (e.g. `@ngrx/*`) in the readme.md file.

Reply:
I removed the `^`'s and left at `10.1.0` because of peer dependencies in `angular/core` - looks like these should be resolved soon with the release of ngrx 11.  From what I've been able to uncover, it appears we would still be safe to `--force`, but i'd prefer to wait until the official ngrx 11 release.

I've updated in the README as well